frontend/src/components/MainPanel.vue

```diff
@@ -28,29 +28,54 @@
 </template>
 
 
-<script>
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
 import Board from './Board.vue'
 import Toolbar from './Toolbar.vue'
+import {toolbarObj} from "../ts/Helpers/toolbar"
 
-export default {
-    name: 'MainPanel',
-    mounted(){
-    },
+@Component({
+  components: {
+    Board,
+    Toolbar,
+  }
+})
 
-    components: {
-        Board,
-        Toolbar,
-    },
-    
-    data: ()=> ({
-        toolbar: null,
-    }),
+export default class MainPanel extends Vue {
```

Comment:
Although it is possibile to write this way components on vue, i think that recommended is to use vue3 way https://v3.vuejs.org/guide/typescript-support.html#editor-support